sycl/include/CL/sycl/item.hpp

```diff
@@ -20,6 +20,8 @@ struct Builder;
 }
 template <int dimensions> struct id;
 template <int dimensions> class range;
+template <int dimensions> struct h_item;
```

Comment:
Probably everything is a `class` instead of a `struct` now

Reply:
ok, will make h_item a class.